test/minter-suite-minters/DA/MinterDAV2.common.ts

```diff
@@ -27,6 +27,122 @@ export const MinterDAV2_Common = async () => {
           value: this.startingPrice,
         });
     });
+
+    describe("payment splitting", async function () {
+      beforeEach(async function () {
+        this.deadReceiver = await deployAndGet.call(
+          this,
+          "DeadReceiverMock",
+          []
+        );
+      });
+
+      it("requires successful payment to platform", async function () {
+        // update platform address to a contract that reverts on receive
+        await this.genArt721Core
+          .connect(this.accounts.deployer)
+          .updateArtblocksPrimarySalesAddress(this.deadReceiver.address);
+        // expect revert when trying to purchase
+        await ethers.provider.send("evm_mine", [
+          this.startTime + this.auctionStartTimeOffset,
+        ]);
+        await expectRevert(
+          this.minter
+            .connect(this.accounts.user)
+            .purchaseTo(this.accounts.additional.address, this.projectZero, {
+              value: this.startingPrice,
+            }),
+          "Art Blocks payment failed"
+        );
+      });
+
+      it("requires successful payment to artist", async function () {
+        // update artist address to a contract that reverts on receive
+        await this.genArt721Core
+          .connect(this.accounts.deployer)
+          .updateProjectArtistAddress(
+            this.projectZero,
+            this.deadReceiver.address
+          );
+        // expect revert when trying to purchase
+        await ethers.provider.send("evm_mine", [
+          this.startTime + this.auctionStartTimeOffset,
+        ]);
+        await expectRevert(
+          this.minter
+            .connect(this.accounts.user)
+            .purchaseTo(this.accounts.additional.address, this.projectZero, {
+              value: this.startingPrice,
+            }),
+          "Artist payment failed"
+        );
+      });
+
+      it("requires successful payment to artist additional payee", async function () {
+        // update artist additional payee to a contract that reverts on receive
+        const proposedAddressesAndSplits = [
+          this.projectZero,
+          this.accounts.artist.address,
+          this.deadReceiver.address,
+          50,
+          this.accounts.additional2.address,
+          50,
```

Comment:
Nit: these two constant values could be helpful to add smol dev comments for.

Reply:
updated 🙏